lib/ansible/modules/system/selinux.py

```diff
@@ -151,6 +151,9 @@ def set_state(module, state):
 
 
 def set_config_policy(module, policy, configfile):
+    if not os.path.exists('/etc/selinux/%s' % policy):
```

Comment:
Could we verify that the file /etc/selinux/%s/policy exist ? While the goal is to prevent errors, someone could use another name (config, final, tmp) that would bypass this check. Not a big deal, but I like to be precise.

Reply:
Good point, fixed :+1: